modules/core/src/main/java/org/apache/ignite/lang/ErrorGroups.java

```diff
@@ -36,12 +43,48 @@ public static class Table {
         public static final ErrorGroup TABLE_ERR_GROUP = ErrorGroup.newGroup("TBL", 2);
 
         /** Table already exists. */
-        public static int TABLE_ALREADY_EXISTS_ERR = TABLE_ERR_GROUP.registerErrorCode(1);
+        public static final int TABLE_ALREADY_EXISTS_ERR = TABLE_ERR_GROUP.registerErrorCode(1);
         /** Table not found. */
-        public static int TABLE_NOT_FOUND_ERR = TABLE_ERR_GROUP.registerErrorCode(2);
+        public static final int TABLE_NOT_FOUND_ERR = TABLE_ERR_GROUP.registerErrorCode(2);
         /** Column already exists. */
-        public static int COLUMN_ALREADY_EXISTS_ERR = TABLE_ERR_GROUP.registerErrorCode(3);
+        public static final int COLUMN_ALREADY_EXISTS_ERR = TABLE_ERR_GROUP.registerErrorCode(3);
         /** Column not found. */
-        public static int COLUMN_NOT_FOUND_ERR = TABLE_ERR_GROUP.registerErrorCode(4);
+        public static final int COLUMN_NOT_FOUND_ERR = TABLE_ERR_GROUP.registerErrorCode(4);
+    }
+
+    /** Meta storage error group. */
+    public static class MetaStorage {
```

Comment:
Why is this a part of public API?

Reply:
I think that all error code should be available for the end user